Samples/AppLifecycle/Instancing/cs-winforms-unpackaged/CsWinFormsInstancing/CsWinFormsInstancing.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
```

Comment:
19041 == 20H1

Do all samples target that? RS5+19H2 are supported but not a target for samples?

Reply:
The C# template still targets 19041 so if not all samples then most of them would still be targeting that. We can update it in the template as well though in the same PR that the .NET version is being update in if we want to do that.